db/warehouse/migrate/20220920192149_add_veteran_override_to_client.rb

```diff
@@ -0,0 +1,6 @@
+class AddVeteranOverrideToClient < ActiveRecord::Migration[6.1]
+  def change
+    add_column :Client, :veteran_override, :boolean, default: false
+    add_column :Client, :va_check_date, :date, default: nil
+  end
```

Comment:
I _think_ we want to store this in a separate table so we can keep a history of checks, and maybe even who requested them.  Seems like:
`client_id, response, check_date, user_id, created_at, updated_at`

I'm not 100% convinced we need it, but only storing the most-recent request, and not storing the full response seems like it might be helpful.